src/vs/workbench/contrib/terminal/browser/media/terminal.css

```diff
@@ -449,6 +449,11 @@
 .terminal-scroll-highlight {
 	left: 0;
 	right: 0;
+	border-left: 5px solid #ffffff;
```

Comment:
don't we normally keep colors out of css?

Reply:
This is just a default, it should never be used in practice unless something bad with the theme happens.